policy.tf

```diff
@@ -1098,3 +1098,181 @@ data "aws_iam_policy_document" "s3_bucket_with_kms_website_policy_document_1" {
   }
 }
 
+data "aws_iam_policy_document" "s3_tls_bucket_policy_document" {
```

Comment:
Instead of duplicating the existing policies and adding the forced TLS stuff to them, you could just have one policy document and policy that only has the forced TLS statement in it and attach that to the users when the flag is set.

Reply:
This has been done